test/test-sets/ref/metrics_.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"cbea_label_first_action":0,"cbea_label_not_first":7,"cbea_labeled_ex":7,"cbea_max_actions":12,"cbea_min_actions":12,"cbea_non_zero_cost":4,"cbea_predict_in_learn":0,"conf_idx_0":0,"conf_idx_1":3,"conf_idx_2":2,"current_champ":0,"line_parse_error":0,"number_events_zero_actions":0,"number_skipped_events":0,"sfm_count_learn_example_with_shared":3,"test_county":3,"total_champ_switches":0,"total_learn_calls":3,"total_log_calls":0,"total_predict_calls":0,"upcnt_0":3,"upcnt_1":2,"upcnt_2":2,"bound_0":0.0,"bound_1":0.0,"bound_2":0.0,"cbea_avg_actions_per_event":12.0,"cbea_avg_feat_per_action":372.0,"cbea_avg_feat_per_event":4471.0,"cbea_avg_ns_per_action":4.0,"cbea_avg_ns_per_event":50.0,"cbea_sum_cost":-4.0,"cbea_sum_cost_baseline":0.0,"dsjson_sum_cost_original":0.0,"dsjson_sum_cost_original_baseline":0.0,"ips_0":0.0,"ips_1":0.0,"ips_2":0.0,"r_0":1.0,"r_1":1.0,"r_2":1.0,"w_0":0.0,"w_1":0.0,"w_2":0.0,"exclusionc_0":"","exclusionc_1":"E: [M ] ","exclusionc_2":"E: [M ] ","first_event_id":"0074434d3a3a46529f65de8a59631939","first_event_time":"","interactions_0":"-q EE -q EG -q EM -q EO -q ER -q ES -q ET -q Ei -q Ej -q E0 -q GG -q GM -q GO -q GR -q GS -q GT -q Gi -q Gj -q G0 -q MM -q MO -q MR -q MS -q MT -q Mi -q Mj -q M0 -q OO -q OR -q OS -q OT -q Oi -q Oj -q O0 -q RR -q RS -q RT -q Ri -q Rj -q R0 -q SS -q ST -q Si -q Sj -q S0 -q TT -q Ti -q Tj -q T0 -q ii -q ij -q i0 -q jj -q j0 -q 00 ","interactions_1":"-q EE -q EG -q EO -q ER -q ES -q ET -q Ei -q Ej -q E0 -q GG -q GM -q GO -q GR -q GS -q GT -q Gi -q Gj -q G0 -q MM -q MO -q MR -q MS -q MT -q Mi -q Mj -q M0 -q OO -q OR -q OS -q OT -q Oi -q Oj -q O0 -q RR -q RS -q RT -q Ri -q Rj -q R0 -q SS -q ST -q Si -q Sj -q S0 -q TT -q Ti -q Tj -q T0 -q ii -q ij -q i0 -q jj -q j0 -q 00 ","interactions_2":"-q EE -q EG -q EO -q ER -q ES -q ET -q Ei -q Ej -q E0 -q GG -q GM -q GO -q GR -q GS -q GT -q Gi -q Gj -q G0 -q MM -q MO -q MR -q MS -q MT -q Mi -q Mj -q M0 -q OO -q OR -q OS -q OT -q Oi -q Oj -q O0 -q RR -q RS -q RT -q Ri -q Rj -q R0 -q SS -q ST -q Si -q Sj -q S0 -q TT -q Ti -q Tj -q T0 -q ii -q ij -q i0 -q jj -q j0 -q 00 ","last_event_id":"9077f996581148978a0ebe2484260dab","last_event_time":""}
```

Comment:
pending: add --debug_metrics to automl reduction to opt-in into these more verbose metrics